hack/util.sh

```diff
@@ -74,6 +74,7 @@ function configure_os_server {
 	openshift start \
 	--write-config=${SERVER_CONFIG_DIR} \
 	--create-certs=false \
+	--dns="tcp://${API_HOST}:53" \
```

Comment:
I'm realizing now the changes to `start_os*` weren't undone by the `oc cluster up` changes